src/main/java/com/bravo/user/model/dto/UserReadDto.java

```diff
@@ -8,5 +8,19 @@
 
   private String id;
   private String name;
+  private Integer phoneNumber;
   private LocalDateTime updated;
+
+  public void setName(String name) {
```

Comment:
These getters/setters are not necessary since the [Lombok](https://projectlombok.org/) `@Data` annotation provides them.